kv/rest_test.go

```diff
@@ -480,8 +480,11 @@ func TestKeysAndBodyArePreserved(t *testing.T) {
 }
 
 func postURL(url string, body io.Reader, t *testing.T) {
-	resp, err := http.Post(url, "text/plain", body)
-	defer resp.Body.Close()
```

Comment:
Reinstate this call to resp.Body.Close(). The go http client tends to leak if you don't explicitly tell it you're done with the body.